src/guides/using-final-ik.md

```diff
@@ -0,0 +1,107 @@
+
+Let's enter play mode and try this out.
+
+<video src="./creating-a-player-controller/sphere-player-mouse-jump-6.mp4" width="100%" controls></video>
+(Video demonstrating the steps above)
+
+As we can see, our character is there, but (TODO: how does it fail?). In order for our character to animate correctly, we'll need to set up an animation controller.
+
+Normcore comes with a built-in animation controller for Final IK. Drag the **PlayerAnimatorController** component from the Unity project into the **Controller** field on the **Animator** component. And to ensure that the **Player** component can drive this animator controller, drag a reference from the "Pilot" game object into the **Character** slot.
+
+<video src="./creating-a-player-controller/sphere-player-mouse-jump-6.mp4" width="100%" controls></video>
+(Video demonstrating the steps above)
+
+Much better! This looks good, but Final IK provides a handful of built-in components that we can use to polish this animation even further.
+
+## Using Final IK components
+
+Our character works well, but there are a few places where it falls short. Our character doesn't lean when running in a circle, and if we add a ramp to the scene that our character can walk up, you can see the character's feet don't line up with the ground.
+
+<video src="./creating-a-player-controller/sphere-player-mouse-jump-6.mp4" width="100%" controls></video>
+(Video demonstrating the steps above)
+
+Luckily, Final IK comes with a variety of components that can be used to refine the animation of our character.
+
+On the "Pilot" game object, add a **FullBodyBipedIK** component, a **BodyTilt** component.
+
+Populate the **BodyTilt** component **IK** field by dragging in a reference to the **FullBodyBipedIK** component. We'll also need to provide a left and right pose. Normcore includes a "Pose Left" and "Pose Right" preset that can be found in the `Normal/Examples/Final IK Player Controller/Character` folder.
+
+You can adjust the values for **Tilt Speed**, and **Tilt Sensitivity** as you like, but we have found 5.0 for Tilt Speed and 0.2 for Tilt Sensitivity look great.
+
+<video src="./creating-a-player-controller/sphere-player-mouse-jump-6.mp4" width="100%" controls></video>
+(Video demonstrating the steps above)
+
+Last, let's add the **GrounderFullBodyBiped** component. Open up the Solver, and set *Layers* to "Default". This is the collision layer used for the solver. As your game grows, you may want to create a dedicated layer for the ground and surfaces you would like your player to land on.
+
+<video src="./creating-a-player-controller/sphere-player-mouse-jump-6.mp4" width="100%" controls></video>
+(Video demonstrating the steps above)
+
+Now that both of those components are set up. Let's test it out! Have the player run in a circle and walk up the ramp.
+
```

Comment:
Its a bit hard to test body tilt without the camera controls in place, since the tilt on walking is hardly noticeable compared to sprinting with follow camera.

Reply:
Fixed in [c8d696f](https://github.com/NormalVR/Normcore-Documentation/pull/16/commits/c8d696fa82d5caebaf36a23ae688e2ffafcaefc4).